src/styles/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function center-position($elementSize) {
+  @return ($stopwatch-size / 2) - $elementSize;
```

Comment:
The variable `$stopwatch-size` is used here but is not defined within this file. Make sure that `$stopwatch-size` is defined or imported from another SCSS file to avoid compilation errors.